django/contrib/contenttypes/admin.py

```diff
@@ -42,8 +40,8 @@ def _check_relation(self, obj, parent_model):
             except FieldDoesNotExist:
                 return [
                     checks.Error(
-                        "'ct_field' references '%s', which is not a field on '%s.%s'." % (
-                            obj.ct_field, obj.model._meta.app_label, obj.model._meta.object_name
+                        "'ct_field' references '%s', which is not a field on '%s'." % (
```

Comment:
I quite like the % format... but should we be considering f-strings here?

Reply:
We don't prefer f-strings over %-formatting, so I don't see a reason to change this just because we're touching this code.